File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    # enter height as input
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
for row in range(height):
    # printing #
    for space in range(height - row - 1, 0, -1):
        print(" ", end="")
    for hash in range(row + 1):
        print("#", end="")
    print("\n", end="")

File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(string prompt);

int main(void)
{
    int number = get_positive_int("Height: ");

    for (int height = 0; height < number; height++) //height and number of lines
    {
        for (int spaces = number - height - 2; spaces >= 0; spaces--) //to print spaces before pyramid
        {
            printf(" ");
        }
        for (int row = 0; row <= height; row++) //left pyramid
        {
            printf("#");
        }
        printf("  ");
        for (int row2 = 0; row2 <= height; row2++) //right pyramid
        {
            printf("#");
        }
        printf("\n");
    }
}

int get_positive_int(string prompt)
{
    int number;
    do
    {
        number = get_int("%s", prompt);
    }
    while (number < 1 || number > 8); //checking the right range between 1 and 8
    return number; //returning if incorrect number
}